main.go

```diff
@@ -131,7 +142,8 @@ func (j junit2jira) createIssueOrComment(tc testCase) error {
 	if err != nil {
 		return fmt.Errorf("could not get description: %w", err)
 	}
-	log.Println("Searching for ", summary)
+	const NA = "?"
```

Comment:
Is this structured logging best practice to provide a sentinel value for irrelevant fields?
Perhaps it would be better to just skip the field altogether?